src/module_1/module_1_meteo_api.py

```diff
@@ -3,34 +3,45 @@
 import logging
 import time
 import sys
-import json
-import numpy as np
+import pandas as pd
 import matplotlib.pyplot as plt
-import pickle 
+import urllib3
+# import json
+# import numpy as np
+
+
+# Disable the InsecureRequestWarning due to using verify=False from corporate laptop
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
 
 """Define global variables"""
+
 API_URL = "https://climate-api.open-meteo.com/v1/climate?"
 
 COOL_OFF_TIME = 5
 MAX_RETRY_ATTEMPTS = 3
 
 COORDINATES = {
-"Madrid": {"latitude": 40.416775, "longitude": -3.703790},
-"London": {"latitude": 51.507351, "longitude": -0.127758},
-"Rio": {"latitude": -22.906847, "longitude": -43.172896},
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
 }
 
 VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
 
-#All models have data for the 3 variables except soil moisture, which is only provided by MRI_AGCM3 and EC_Earth
-MODELS = "CMCC_CM2_VHR4,FGOALS_f3_H,HiRAM_SIT_HR,MRI_AGCM3_2_S,EC_Earth3P_HR,MPI_ESM1_2_XR,NICAM16_8S"
+# All models have data for the 3 variables except soil moisture
+# which is only provided by MRI_AGCM3 and EC_Earth
+MODELS = "CMCC_CM2_VHR4,FGOALS_f3_H,HiRAM_SIT_HR,MRI_AGCM3_2_S,EC_Earth3P_HR,MPI_ESM1_2_XR,NICAM16_8S" # noqa
+
 
 """Define auxiliar functions"""
+
 def call_api(url):
-#Need to add the verify=False as working from my corporate laptop. Tried to authenticate SSL by changing
-#lots of different things but issue still arises. Disabled SSL but only works from the office - not the best practice I know
+    # Need to add the verify=False as working from my corporate laptop.
+    # Tried to authenticate SSL by changing lots of config, disabled SSL, etc
+    # It only works from the office - not the best practice I know # noqa
     try:
-        #to-do: add the cool off
+        # to-do: add the cool off
         response = requests.get(url, verify=False)
         if response.status_code == 200:
             print("\nConnected Succesfully!")
```

Comment:
Any reason to use new line at the beginning of the print / log? I think at the end makes more sense

Reply:
Don't mix prints and loggins. You can use `logging.info` instead of "prints" and redirect the output to the console

Reply:
1. Initially used \n at the beginning to separate logs from the first warning triggered by SSL verify=false. Once it has been disabled, you're right - no need to use it here.
2. Solved. I've configured the logging system just before the function for that (now I receive the logs in the shell :D). It is set to `level=logging.INFO`. I understand that "DEBUG" is for max. verbosity. What is the common value used there when deployed to PROD?

Reply:
Yes, exactly, DEBUG is the max verbosity. Normally in production one uses `INFO` but this parameter is normally parametrised in a `yaml` file so can be changed easily between executions / environments to a desired level. For example, if one wants to solve a problem, can change it to `DEBUG`